canonical/identity-platform-admin-ui
# WD-13860 - feat: return to URL that initiated login

## changes

- Set the `next` path when logging in.
- Redirect to the `next` path when loading the UI.

https://warthogs.atlassian.net/browse/WD-13860

## QA:

- Navigate to a page and remember which one you're on.
- Open http://localhost:8000/api/v0/auth/logout in a new tab.
- Go back to the UI and refresh the page and you should see the login button.
- Click the login button and you should get logged in and then returned back to the UI on the page you were on previously.
